.github/workflows/core.yml

```
@@ -892,18 +892,70 @@ jobs:
+        with:
+          submodules: recursive
+          persist-credentials: false
+      - uses: ./.github/actions/environment
+      - name: Generate translation blobs
+        run: nix-shell --run "uv run make -C core translations"
+      - uses: actions/download-artifact@70fc10c6e5e1ce46ad2ea6f2b72d43f7d47b13c3  # actions/download-artifact@v8.0.0
+        with:
+          pattern: core-firmware-*-*-normal
+          path: downloaded-artifacts
+      - name: Detect version
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2006:style:1:9: Use $(...) notation instead of legacy backticks `...`

(shellcheck)